pkg/eval/platform/platform.go

```diff
@@ -35,9 +37,46 @@ import (
 // Whether or not the platform is Microsoft Windows.
 // This is read-only.
 
+//elvdoc:fn hostname &omit-domain=false
+//
+// The host name of the system. If option `&omit-domain` is used any domain
+// components are stripped from the host name. If an error occurs while
```

Comment:
You documented the option as "any domain components are stripped from the host name"; `&strip-domain` would be more appropriate.

It's also better to document explicitly what is considered the domain name: anything after the first dot.

Reply:
Tomato, Tomahto :-) I think `omit-domain` is perfectly clear but it's not worth arguing. Done.

Good point about explaining what it means to omit or strip the domain. Done.